source/release-notes/1.8.txt

```diff
@@ -120,6 +75,55 @@ Issues Fixed:
   ``/progress`` endpoint to potentially return incorrect 
   ``totalBytes`` for sharded clusters.
 
+{+c2c-full-beta-program+}
```

Comment:
Can we have a larger line break or space before this section to make it easier to see?

Reply:
updated: opted for different header.